scripts/update_timestamp_explicit_client.sql

```diff
@@ -0,0 +1,13 @@
+-- UPDATE SCRIOT FOR CLIENT SCHEMA
+-- This script will set any null UpdateTimes to now
```

Comment:
As we use `UpdateTime` for decisions about how recent a record is, setting NULLs to now has the potential to cause problems.

I think it would be better to set any NULLs (which there shouldn't be any of hopefully) to '0000-00-00 00:00:00' (the “zero” timestamp) in all the update scripts. Thoughts?